aspnetcore/tutorials/min-web-api.md

```diff
@@ -214,18 +226,47 @@ This tutorial uses [Endpoints Explorer and .http files](xref:test/http-files#use
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-This tutorial uses Postman to test the API.
+## Create API testing UI with Swagger
+
+To test the API, this tutorial utilizes the .NET package [Swashbuckle.AspNetCore](https://www.nuget.org/packages/Swashbuckle.AspNetCore), which integrates Swagger tools for generating a testing UI adhering to the OpenAPI specification:
```

Comment:
I like this approach. You might also say something about this being one option among many. Maybe something like this?

```suggestion
To test the API, you can choose from many available tools. If you don't have a preferred tool, this tutorial uses utilizes the .NET package [Swashbuckle.AspNetCore](https://www.nuget.org/packages/Swashbuckle.AspNetCore), which integrates Swagger tools for generating a testing UI adhering to the OpenAPI specification:
```

Reply:
Got this suggestion in, but I commited a rewrite for NSwag, and then realized later that I had not commited your suggestion beforehand, so I added it after.  Sorry, didn't mean to miss getting your commit in.